lib/src/main/java/com/diffplug/spotless/FeatureClassLoader.java

```diff
@@ -74,4 +77,20 @@
 		return super.findClass(name);
 	}
 
+	/**
+	 * Making spotless Java 9+ compatible. In Java 8 (and minor) the bootstrap
+	 * class loader saw every platform class. In Java 9+ it was changed so the
+	 * bootstrap class loader does not see all classes anymore. This might lead
+	 * to ClassNotFoundException in formatters (e.g. freshmark).
+	 *
+	 * @return <code>null</code> on Java 8 (and minor), otherwise <code>PlatformClassLoader</code>
+	 */
+	@Nullable
+	private static ClassLoader getParentClassLoader() {
+		try {
+			return (ClassLoader) ClassLoader.class.getMethod("getPlatformClassLoader").invoke(null);
```

Comment:
IMO, feature-testing and version-parsing are roughly equivalent in terms of reliability and ease of maintenance.  I'm happy to accept either, PR author's choice.

Reply:
Sadly version parsing is not that easy. See e.g. Gerade's [Java Version](https://github.com/gradle/gradle/blob/master/subprojects/base-services/src/main/java/org/gradle/api/JavaVersion.java). If we just stick to the major version it could be doable with little effort. I will check tomorrow what's better.